.env.development

```diff
@@ -18,6 +18,8 @@ NEXT_PUBLIC_PUBLIC_MAIL="defuse@defuse.org"
 NEXT_PUBLIC_LINK_X="https://x.com/DefuseProtocol"
 NEXT_PUBLIC_LINK_DISCORD="https://discord.gg/G6Eh8Z3M"
 NEXT_PUBLIC_LINK_DOCS="https://docs.google.com/document/d/1yK5zjZpdA8IjiGc3JSzb7dfyt-5qsBjMMMi4KbYNHkM/edit?usp=sharing"
+NEXT_PUBLIC_SENTRY_ENABLED="true"
```

Comment:
Probably don't want it enabled in development. Otherwise it will get polluted with errors while people are developing stuff! Anyway we can set this as a Vercel env var, maybe leave it out of all of these files

Reply:
Right, I see reasonable move it to Vercel env.